modules/installing-ocp-agent.adoc

```diff
@@ -0,0 +1,170 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent_{context}"]
+= Installing {product-title} with Agent
+
+You can use the Agent-based installer to install a:
+
+* Single-node cluster
+* Compact cluster
+* Highly Available (HA) cluster
```

Comment:
Not sure if the prerequisites for these 2 cluster topologies need to be added. There are here for ref https://github.com/openshift-metal3/dev-scripts/blob/master/common.sh#L369-L392

Reply:
I think we should add the prerequisite details for all the cluster topologies we support.

Reply:
Let me check and get back but it's a valid point. Need to check if the pre-req for these topologies are documented else in docs.openshift